install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo "Creating venv..."
```

Comment:
If we want to do this, we should add `venv` to the requirements. In some systems (Ubuntu/Debian for example) it's not installed by default.

Reply:
I think this is the best practice, but this is just my suggestion.